internal/oci/uvm.go

```diff
@@ -305,6 +307,25 @@ func parseAnnotationsUint64(ctx context.Context, a map[string]string, key string
 	return def
 }
 
+// parseAnnotationsInt32 searches `a` for `key` and if found verifies that the
+// value is a 32 bit signed integer. If `key` is not found returns `def`.
+func parseAnnotationsInt32(ctx context.Context, a map[string]string, key string, def int32) int32 {
```

Comment:
If the `VirtualNodeCount` is never going to be negative then why do we need this separate function? We already have the `parseAnnotationsUint32` function for unsigned values. Maybe just make the `VirtualNodeCount` field a uint32? Just curious.

Reply:
The schema has it generated as an int32 and this option isn't LCOW/WCOW specific so it's on the actual options struct not a field on optionsLCOW/WCOW. I just didn't want to do a type conversion when passing it in and a type conversion on the returned value afterwards. I suppose we could change the actual schema value to be uint32 (seeming as how it actually is a uint before getting generated. uints get generated as signed integers with our schema for some reason but that's another problem).

Reply:
Makes sense. Thanks.